customer-terminate-service/app/src/main/java/se/inera/intyg/cts/infrastructure/service/SendPackageNotification.java

```diff
@@ -0,0 +1,158 @@
+package se.inera.intyg.cts.infrastructure.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import se.inera.intyg.cts.domain.model.Termination;
+import se.inera.intyg.cts.domain.model.TerminationId;
+import se.inera.intyg.cts.domain.repository.TerminationRepository;
+import se.inera.intyg.cts.domain.service.SendNotification;
+import se.inera.intyg.cts.infrastructure.integration.SendEmail;
+import se.inera.intyg.cts.infrastructure.integration.SendSMS;
+
+@Service
+public class SendPackageNotification implements SendNotification {
```

Comment:
I think you should move any changes to the domain object out of this service. This service purpose is to send notifications. It should leave it to the domain to make necessary updates to the domain objects itself.